.github/workflows/wheel-builder.yml

```diff
@@ -116,3 +116,35 @@ jobs:
         with:
           name: "bcrypt-${{ github.event.client_payload.BUILD_VERSION }}-${{ matrix.WINDOWS.WINDOWS }}-${{ matrix.PYTHON.VERSION }}"
           path: bcrypt-wheelhouse\
+
+  manylinux-arm64:
+     name: "Python ${{ matrix.PYTHON }} for manylinux2014-aarch64"
+     runs-on: ubuntu-latest
+     strategy:
+       matrix:
+         PYTHON: ["cp36-cp36m"]
```

Comment:
Last request -- this should be cp35-cp35m to support all the py3 versions we support.

Reply:
Updated and amended. Please check.